runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -3887,6 +3893,7 @@ TR_RelocationRecordSymbolFromManager::preparePrivateData(TR_RelocationRuntime *r
    {
    TR_RelocationSymbolFromManagerPrivateData *reloPrivateData = &(privateData()->symbolFromManager);
 
+   uint16_t type = reloTarget->loadUnsigned8b(&_record->_type);
```

Comment:
What is the purpose of this?

Reply:
It's a remnant of the old way of checking for discontiguous relocations... I meant to remove it before submitting this PR, but I guess I forgot. :sweat_smile: